tests/std/tests/P2374R4_views_cartesian_product/test.cpp

```diff
@@ -973,6 +973,14 @@ namespace check_recommended_practice_implementation { // MSVC STL specific behav
     STATIC_ASSERT(sizeof(range_difference_t<cartesian_product_view<all_t<Vec>, all_t<Vec>>>) > sizeof(ptrdiff_t));
 } // namespace check_recommended_practice_implementation
 
+constexpr void test_gh_3733() {

```

Comment:
Code tends to outlive metacode databases; it's likely this test will exist long after "GH-3733" loses meaning. We conventionally try to include a one-line summary description of the issue in regression tests, something like "`cartesian_product_view` would incorrectly reject a call to `size` claiming that `big*big*big*0` is not representable because `big*big` is not."